cmd/plugin-backend.go

```diff
@@ -26,7 +26,7 @@ var (
 	corsMaxAge  = flag.String("cors-max-age", "", "CORS allowed max age (default: unset)")
 	// todo: default value temporarily kept to make it work with older versions of the NOO. Remove default and force setup of loki url
 	lokiURL      = flag.String("loki", "http://localhost:3100", "URL of the loki querier host")
-	lokiLabels   = flag.String("loki-labels", "SrcNamespace,SrcWorkload,DstNamespace,DstWorkload,FlowDirection", "Loki labels, comma separated")
+	lokiLabels   = flag.String("loki-labels", "SrcK8S_Namespace,SrcK8S_OwnerName,DstK8S_Namespace,DstK8S_OwnerName,FlowDirection", "Loki labels, comma separated")
```

Comment:
Sorry I must have missed the related discussion. Why this change?

Reply:
this is fundamentally the same thing ("Workload" and "Owner" mean the same, it's whatever controls the pods).
The renaming is a consequence of FL2M mechanics for kube enrichment where enriched fields are named `Prefix + "_" + base name` ; the prefix is defined here: https://github.com/netobserv/network-observability-operator/pull/60/files#diff-4d2762740f523111148b4e0e1f03cd3d907d3de8ea2dfc9d40fc194efabadd70R270-R277
We could remove the `K8S` part, but I don't finding it annoying, and it helps clarify which are enriched fields and which aren't.

Reply:
Ok, thanks for the explanation! Anyway we can always adapt FLP to use another enrichment format, if you consider it's needed.

Reply:
(Actually, I was integrating the transformer from Goflow-Kube so the outputs of both versions would be the same)